src/main/java/slimeknights/tconstruct/common/config/Config.java

```diff
@@ -329,6 +330,14 @@ public static boolean syncConfig() {
       propOrder.add(prop.getName());
       Util.setTemperaturePref(temperatureCelsius);
 
+      prop = configFile.get(cat, "minFluidHeight", minFluidHeight);
+      prop.setComment("Minimum fluid height to display in smeltery. Make those tiny amounts easier to click on. Can make the Smeltery appear more full than it really is.");
+      prop.setMinValue(1);
```

Comment:
I'd actually keep 2 or 3 as the minimum value. 1 Simply isn't really usable, as you said yourself.